### PR TITLE
Trans pride version

### PR DESCRIPTION
**Pull request by emckee2006**

### CODE REVIEW

File: sudokupad-bet-tree.user.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // ==UserScript==
 // @name         SudokuPad Bet Tree
 // @namespace    http://tampermonkey.net/
-// @version      0.5
+// @version      0.4
 // @description  Add a bet tree control to SudokuPad
 // @author       Chameleon
-// @updateURL    https://github.com/yusitnikov/sudokupad-bet-tree/raw/main/sudokupad-bet-tree.user.js
+// @updateURL    https://github.com/emckee2006/sudokupad-bet-tree/raw/main/sudokupad-bet-tree.user.js
```
Comment (yusitnikov) [inline sudokupad-bet-tree.user.js:7]: I can't merge the PR with a modified `updateURL`.

File: sudokupad-bet-tree.user.js [hunk 2]
```diff
@@ -17,130 +17,15 @@
 window.addEventListener('DOMContentLoaded', () => {
 	const treeMarker = 't';
 
-	const renderTree = (renderer, scaleCoeff, offsetX, offsetY) => {
```
Comment (yusitnikov) [inline sudokupad-bet-tree.user.js:20]: The job of this function was to ensure that the bet tree itself and the button's icon look exactly the same, and I'd like to leave it this way.

**Review by yusitnikov — CHANGES_REQUESTED** [review]
Hi @emckee2006 , good work!

I know that several people are already using this extension, and I don't want to force them to use the pride version. So if you want this PR to be merged, please make the button configurable. Of course, merging the PR is not a must, because people could just install the themed script from your fork instead of mine.

**Comment by yusitnikov** [comment]
P.S. The merge conflicts are because I just removed the Christmas theming. I guess that it's the time :D

**Comment by emckee2006** [comment]
That's ok... I made it because Zeta was wanting one of those to replace
Christmas... How would I make it configurable?

**Comment by yusitnikov** [comment]
> That's ok... I made it because Zeta was wanting one of those to replace Christmas... How would I make it configurable?

~~You can use the `Framework.addSetting` API to add a new toggle or select-box to the settings. See https://sudokupad.app/tool-letter.js for example.~~
I added a framework for themes in the main branch. Add the selectbox value/label to `themeSetting.options` and the renderer implementation to `themes`.

**Comment by yusitnikov** [comment]
And now the pride theme is in too, so I'm closing the PR. Please write here or open an issue if there's something wrong with it

**Pull request closed without merging**